var/spack/repos/builtin/packages/py-waves/package.py

```diff
@@ -21,14 +21,18 @@ class PyWaves(PythonPackage):
     license("BSD-3-Clause", checked_by="kbrindley")
 
     version("main", branch="main", get_full_repo=True)
+    version("0.13.1", sha256="3d776ebd07d05e7de8704e4491864336cab4ee23c284106056a805bf7561435a")
+    version("0.13.0", sha256="cf7072ffc9a08789dc764d2beb3a1a6762a034c954c77fe056d67b4f14272f17")
+    version("0.12.10", sha256="f620f8b7c01487bf387d52312c12da8afe8ed8fbf97b0d488525cdfcb820356e")
+    version("0.12.9", sha256="876528cc0c9ecd73fc5fa780ed4c6a3305865a6cf587dff5fbe78f2801a9eceb")
     version("0.12.8", sha256="6055b4e92075c0168bf9d85b75375f2f40fa37cd3be2dce80a9b304f3523dc81")
     version("0.12.7", sha256="53d4f389351b89eb9f852f7a999a9119384aa9b62a21c27920de006f37a2e7c4")
     version("0.12.6", sha256="f25ec014b04319512d4227f7f6fb788056de60e5686795d9b5c189f1cc61b7f2")
     version("0.12.5", sha256="3868f1592a21e4b671ed31e66951151d73ff0535e0209c9621629994b25c0cd4")
 
     depends_on("python@3.9:", type=("build", "run"))
 
-    depends_on("git", when="@develop", type="build")
+    depends_on("git", when="@main", type="build")
```

Comment:
I don't think we should need to depend on git here. Spack takes care of the git dependency when using git as a fetch method.